docs/schedule-exceptions.adoc

```diff
@@ -0,0 +1,82 @@
+= Schedule exceptions
+
+Schedule exceptions describes any deviation from “regular schedule” generated from KOS data, i.e. state holiday, dean’s holiday, temporary change of room etc.
+Exceptions are applied to events (i.e. modifies its attributes) during automatic planning process.
+
+When an event is affected by an exception, then you will know about it from the attribute `links.applied_exceptions` in the event.
+One event may be affected by multiple exceptions.
+
+.Attributes:
+* `id` – unique numeric ID
+* `type` – see <<Types>>
+* `name` – name for users
+* `note` – any additional information for users
+* `is_regular` –  TODO
+* `scope` – see <<Scope>>
+* `options` – see <<Types>>
+
+
+== Types
+
+We have multiple types of exceptions for various situations.
+This list is not final, we will add more types in the future.
+
+=== CANCEL
+
+Event is cancelled without a substitute.
+
+Attribute `deleted` is set to `true`, i.e. the event is not included in events resources, unless URI parameter `deleted=true` is used.
+
+=== RELATIVE_MOVE
+
+Event is shifted by a time offset (positive, or negative).
+
+The specified offset is added to `starts_at` and `ends_at`, original values are stored in `original_data.starts_at` and `original_data.ends_at`.
+
+.Attributes:
+* `options.offset` – offset in minutes
+
+=== ROOM_CHANGE
+
+Event is moved to another room.
+
+The event’s `room_id` is changed to the specified ID and original value is stored in `original_data.room_id`.
+
+.Attributes:
+* `options.room_id` – KOS code (https://rozvoj.fit.cvut.cz/Main/znaceni-mistnosti[anatomy of KOS code])
+
+
+== Scope
+
+Scope is used to narrow a set of events that should be affected by the given schedule exception.
```

Comment:
_narrow a set_ seems strange. I understand your intent, but it just doesn’t work like that. How about _specify_ or _define_.

> Scope is used to **define** a set of events that should be affected by the given schedule exception.